### ISSUE TITLE
Python 2.7 test command is broken

### ISSUE THREAD

**Issue by distributist** [buildout/buildout#661]
When running make PYTHON_VER=2.7 test, the command exits with an error:
```
make: .../pythons/2.7/bin/virtualenv: No such file or directory
make: *** [Makefile:115:../venvs/2.7/bin/python2.7] Error 127

```
This can be fixed by changing line 115 in Makefile from:
`$(PYTHON_PATH)/bin/virtualenv -p $(PYTHON_PATH)/bin/$(PYTHON_EXE) $(VENV)`
to:
`$(PYTHON_PATH)/bin/$(PYTHON_EXE) -m virtualenv -p $(PYTHON_PATH)/bin/$(PYTHON_EXE) $(VENV)`

This seems to work fine when using Python 3.x as well.

**Comment by mauritsvanrees** [comment]
If that works for you, it is fine with me.  In CI the current code [works fine](https://github.com/buildout/buildout/actions/runs/10835456784/job/30067132738), though I am not completely sure the Makefile is used there.
The current code fails for me with `zipimport.ZipImportError: can't decompress data; zlib not available` but that may be something on my Mac.

Can you create a PR?

I want to drop 2.7 support though, probably this year still.  And simplify the testing.  But as long as fixes for 2.7 are easy, it would be nice to keep.  I would like to test the 3.1.0 release a bit longer in the wild before creating a 3.x maintenance branch and possibly needing to backport fixes.

**Comment by mauritsvanrees** [comment]
You PR was merged, so closing this issue.
